recipes/pennylane-lightning-gpu/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "pennylane-lightning-gpu" %}
+{% set version = "0.29.0" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # - url: https://github.com/PennyLaneAI/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  #   sha256: e4cb8c496bddbb6c6cd27a6e1462cd3625952996222185dd0d7ec82e3cafea9b
+  - git_url: https://github.com/PennyLaneAI/pennylane-lightning-gpu.git
+    git_branch: bugfix/CUSPARSE_MV_ALG_DEFAULT
```

Comment:
Please use a long-lived, official tag/release. If there's any need to patch for packaging purpose, you can add the patches here.
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=rpath#patches